Tutorials/ACCESS-NRI_Intake_Catalog.ipynb

```diff
@@ -4,16 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Do we need to note which NCI projects are required? Currently, we still need xp65 to access the master catalogue ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good point!

Reply:
done via [d3fbc6e](https://github.com/COSIMA/cosima-recipes/pull/404/commits/d3fbc6e9f4e612fee3841c4e54defc38dcaf63b7)